module_2/Singelton/src/Superman.php

```diff
@@ -0,0 +1,27 @@
+<?php
+class Superman
```

Comment:
@Hlopec It make sense to wrap a Singleton into a specific trait and reuse it where it's necessary.